uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
```

Comment:
I'm inclined to agree. This (and the export) may even need to go in its new section. I suppose there's no way to make a specific function/page more restrictive than the class?